kimchi/js/bindings.js

```diff
@@ -1866,6 +1857,152 @@ function linearization_of_rust(linearization, affine_class) {
 // Provides: None
 var None = 0;
 
+// Provides: caml_is_none
+// Requires: None
+var caml_is_none = function (v) {
+  return v === None;
+};
+
+// Provides: caml_map_of_rust_vector
+// Requires: js_class_vector_of_rust_vector
+var caml_map_of_rust_vector = function (v, klass, converter_to_rust) {
+  var a = js_class_vector_of_rust_vector(v, klass);
+  var res = [0];
+  for (var i = 0; i < a.length; ++i) {
+    // TODO Check this. Could be off by 1
+    res.push(converter_to_rust_rust(a[i]));
+  }
+  return res;
+};
+
+// Provides: caml_opt_of_rust
+// Requires: None
+var caml_opt_of_rust = function (value, value_of_rust) {
+  if (value == undefined) {
+    return None;
+  } else {
+    return [0, value_of_rust(value)];
+  }
+};
+
+// Provides: caml_bool_of_rust
+var caml_bool_of_rust = function (bool) {
```

Comment:
Not sure it is correct. I would say simply `bool`: https://rustwasm.github.io/wasm-bindgen/reference/types/bool.html.